tests/User/Auth0UserIdentityResolverTest.php

```diff
@@ -87,7 +87,7 @@ public function it_gets_user_by_email()
 
         $users->expects($this->atLeast(1))
             ->method('getAll')
-            ->with(['q' => 'email:"ivo@hdz.com"'])
+            ->with(['q' => 'email:"'.urlencode("ivo@hdz.com").'"'])
```

Comment:
Nit: In tests, I like hardcoding the expected value instead of running it through the same conversion functions. Then the test is separated from the implementation and you know for sure the output is what you'd expect it to be. Of course in this case `urlencode` should work fine as it's a built-in function and its usage is straightforward, so we can leave this as-is if you want. It's more of an issue with more complicated functions where we might actually be using the wrong options / flags / arguments and we repeat that mistake in the test.

Reply:
Valid point, I will update it :)